libbeat/outputs/tidb_slow_query/client.go

```diff
@@ -168,6 +168,12 @@ func getFields(event publisher.Event) []interface{} {
 	for _, c := range orderedColumn {
 		// expect nil if fields not exist
 		v, _ := event.Content.GetValue(c)
+		// fixme: there is and inconsistency between slow log and CLUSTER_SLOW_QUERY table schema
+		// fixme: "User@Host" in logs   ---->    "User" in table
+		// fixme: hard code here
```

Comment:
I think it's just fine to display user@host

Reply:
This change aims to fix missing the `User` field. If slow logs are from 5.2.0 TiDB clusters, there is no `# User: ` line in the record, which is replaced by `# User@Host: ` line. Thus I have to get values from both `# User: ` and `# User@Host: `, then save the value to `User` column in the schema.